### PR TITLE
Merge 1.4 into 1.5

### PR DESCRIPTION
**Pull request by solivier**
| Q                                 | A
| --------------------------------- | ---
| Review and 2 GTM                  |

### CODE REVIEW

**Comment by grena** [comment]
GTM

File: web/app.php [hunk 1]
```diff
@@ -1,24 +1,18 @@
 <?php
 
-use Symfony\Component\ClassLoader\ApcClassLoader;
```
Comment (SamirBoulil) [inline web/app.php]: Can you explain the changes in here ? is this normal ?

Comment (solivier) [inline web/app.php]: the changes are explained in the related PR coming from 1.4 https://github.com/akeneo/pim-community-dev/pull/4957 here it's just a merge of fixes from 1.4 to 1.5

**Comment by momoss** [comment]
GTM

**Pull request merged**